materialize-bigquery/bigquery.go

```diff
@@ -76,7 +57,7 @@ type tableConfig struct {
 	base *config
 
 	Table string `json:"table" jsonschema:"title=Table,description=Table in the BigQuery dataset to store materialized result in."`
-	Delta bool   `json:"delta_updates,omitempty" jsonschema:"default=true,title=Delta Update,description=Should updates to this table be done via delta updates. Defaults is false."`
+	Delta bool   `json:"delta_updates,omitempty" jsonschema:"default=false,title=Delta Update,description=Should updates to this table be done via delta updates. Defaults is false."`
```

Comment:
Is this actually changing the default, or was this descriptive of the current behavior?

Reply:
@psFried the `default` in JSONSchema was not working, and in Golang the default value for bool is `false`. The description also used to say that it defaults to false.